prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,1155 @@
+        "route53_domains_privacy_protection_enabled",
+        "route53_domains_transferlock_enabled"
+      ]
+    },
+    {
+      "Id": "Ban outbound DNS calls from all VPCs to ports 53",
+      "Description": "Prohibit all Virtual Private Clouds (VPCs) from initiating outbound DNS calls on port 53.",
+      "Attributes": [
+        {
+          "Section": "R53 DNS Resolver Firewall",
+          "Service": "Route53"
+        }
+      ],
+      "Checks": [
+        "route53_domains_privacy_protection_enabled",
+        "route53_domains_transferlock_enabled"
+      ]
+    },
+    {
+      "Id": "Apply custom threat list for GuardDuty to alert on access to DoH servers",
+      "Description": "Implement a customized threat list within GuardDuty to generate alerts when there is access to Domain Name System over HTTPS (DoH) servers.",
+      "Attributes": [
+        {
+          "Section": "R53 DNS Resolver Firewall",
+          "Service": "Route53"
+        }
+      ],
+      "Checks": [
+        "route53_domains_privacy_protection_enabled",
+        "route53_domains_transferlock_enabled"
+      ]
+    },
+    {
+      "Id": "Enable Shield Advanced subscription for public facing account",
+      "Description": "Activate the Shield Advanced subscription for the publicly accessible account.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "Shield"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
+      ]
+    },
+    {
+      "Id": "Export metrics in centralized collector",
+      "Description": "Exporting metrics to a centralized collector for data aggregation and analysis.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "CloudWatch"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
+      ]
+    },
+    {
+      "Id": "Create DDoS battle card with main info about protected services",
+      "Description": "Prepare a Detailed Distributed Denial of Service (DDoS) Battle Card Encompassing Key Information Regarding Safeguarded Services.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "Shield"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
```

Comment:
Please add `shield_advanced_protection_in_associated_elastic_ips` and `shield_advanced_protection_in_global_accelerators`

Reply:
Done!